src/duplicacy_gcdstorage.go

```diff
@@ -45,11 +46,19 @@ type GCDConfig struct {
 }
 
 func (storage *GCDStorage) shouldRetry(threadIndex int, err error) (bool, error) {
+	const LIMIT_BACKOFF_TIME = 64
```

Comment:
This whole 6-line block looks like it could be pulled out as member variables (or consts), because they don't appear to change between method calls. That would avoid spending a few CPU cycles on each method call, but also slim down the method body to focus on the actual algorithm.